src/main/java/core/basesyntax/serviceparser/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.serviceparser;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitTransactionParserImpl implements FruitTransactionParser {
+    @Override
+    public List<FruitTransaction> parse(List<String> lines) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : lines) {
+            String[] parts = line.split(",");
+            if (parts.length != 3 || "type".equals(parts[0])) {
+                continue;
+            }
+
+            FruitTransaction.Operation operation
+                    = FruitTransaction.Operation.valueOf(parts[0].toUpperCase());
+            String fruit = parts[1];
+            int quantity = Integer.parseInt(parts[2]);
```

Comment:
Consider adding error handling for `Integer.parseInt(parts[2])` to manage potential `NumberFormatException` if the quantity is not a valid integer.